fecgov/fec-cms
# Rename strategic plan OIG category

## Summary (required)

- Resolves #4860 

Update strategic plan for OIG so that it is unique and doesn't conflict with our other strategic plans.

### Required reviewers

1 front end

## Impacted areas of the application

General components of the application that this PR will affect:

- OIG landing page reports feed: http://localhost:8000/office-inspector-general/
- OIG reports feed: http://localhost:8000/about/reports-about-fec/oig-reports/

## Screenshots

<img width="497" alt="Screen Shot 2021-09-07 at 10 32 10 AM" src="https://user-images.githubusercontent.com/12799132/132362811-3ab98db8-f8bd-41a3-9e9f-fda9a59f5ff7.png">

<img width="896" alt="Screen Shot 2021-09-07 at 10 35 07 AM" src="https://user-images.githubusercontent.com/12799132/132363258-050e29b3-4737-4a52-ae39-c3f72c128a91.png">


## Related PRs

Related PRs against other branches:

branch | PR
------ | ------
feature/4099-oig-landing-page | [link](https://github.com/fecgov/fec-cms/pull/4311)

## How to test

- checkout this branch
- `./manage.py migrate` - run migrations since this required a new migration file
- `./manage.py runserver`
- Go to the OIG reports feed in Wagtail and update any report to have the "Strategic plan - OIG". Make sure that the OIG strategic plan now appears on the landing page of OIG and on their OIG reports feed page as that category.
